src/js/angular/rest/repositories.rest.service.js

```diff
@@ -84,7 +85,7 @@ function RepositoriesRestService($http) {
     }
 
     function getPrefix(repositoryId, params) {
-        return $http.post(`${REPOSITORIES_ENDPOINT}/${repositoryId}/prefix`, null, { params })
+        return $http.post(`${REPOSITORIES_ENDPOINT}/${repositoryId}/prefix`, null, {params});
```

Comment:
REPOSITORIES_ENDPOINT has value 'res/repositories', the endpoint you need is 'repositories'. Now there is an exception.

Reply:
hmm why is this difference? Is that they actually override the yasqe's request?

Reply:
You are correct and this is the wrong service for this function. It should be in the `RDF4JRepositoriesRestService` where are calls to the correct endpoint.

Reply:
@svilenvelikov the `rest/repositories` is the GDB repositiory controller which deals with repo management. It was my mistake to use this service. The one that deals with the repository data is `repositories/*` endpoint, which is used by `RDF4JRepositoriesRestService`